data/client/king_phisher/client_config.json

```diff
@@ -5,6 +5,9 @@
   "mailer.max_messages_per_connection": 5,
   "mailer.message_type": "email",
   "mailer.target_type": "file",
+  "menu_item_expired": false,
```

Comment:
I'd like to see all these names changed and with a meaningful prefix. After this PR lands "menu_item_expired" will be a bit difficult to remember. Something like "filter.campaign.expired", "filter.campaign.other_users", etc. I think would be easier to remember.